src/components/expression/expression-renderer.js

```diff
@@ -342,7 +423,7 @@ genotet.ExpressionRenderer.prototype.drawMatrixCells_ = function() {
     .style('fill', colorScale)
     .on('mouseover', function(value, i, j) {
       var hoverCell = d3.event.target;
-      var cell = this.cell_ = {
+      var cell = genotet.ExpressionRenderer.Cell = {
```

Comment:
Wrong usage. You should not be using writing typedef-ed class name anywhere other than your declaraion.

Also what you do here is dangerous. You are passing object reference to a new object. What's the purpose? If you want to initialize it to empty, then you should create a constructor class for this object.
```js
genotet.ExpressionRenderer.Cell = function() {
// ...
};
var cell = new genotet.Expressionrenderer.Cell();
_(cell).extend({
  // ...
});
```